src/test/java/hu/akarnokd/rxjava2/parallel/ParallelOrderedMergeTest.java

```diff
@@ -0,0 +1,23 @@
+package hu.akarnokd.rxjava2.parallel;
+
+import java.util.SplittableRandom;
+
+import org.junit.Test;
+
+import io.reactivex.Flowable;
+import io.reactivex.functions.Function;
+import io.reactivex.schedulers.Schedulers;
+
+public class ParallelOrderedMergeTest {
+  @Test
+  public void testBasicOrder() throws InterruptedException {
+    final SplittableRandom rnd = new SplittableRandom();
```

Comment:
Please use Java 6 compatible components in [tests](https://travis-ci.org/akarnokd/RxJava2Extensions/builds/292108662#L502).